src/style.css

```
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
```

Use the same font-family for both font-faces
```suggestion
@font-face {
  font-family: Avenir;
  src: url(fonts/Avenir-Book.ttf);
  font-weight: 300;
}

@font-face {
  font-family: Avenir;
  src: url(fonts/Avenir-Heavy.ttf);
  font-weight: 900;
}
```